rpcd/playbooks/setup-logging.yml

```diff
@@ -14,6 +14,8 @@
 # limitations under the License.
 
 - include: elasticsearch.yml
+  when:
+    - not lookup('env', 'UPGRADE_ELASTICSEARCH') | bool
```

Comment:
this variable needs something in docs so users know what this is and why they might want to set it.

Reply:
Yeah, I plan to wrap all the instruction and docs when we approve this.

Reply:
Still needs docs